class.jetpack-gutenberg.php

```diff
@@ -284,13 +283,15 @@ public static function get_preset( $preset ) {
 	 * @return array A list of blocks: eg [ 'publicize', 'markdown' ]
 	 */
 	public static function jetpack_set_available_blocks( $blocks ) {
-		$preset_blocks_manifest =  self::preset_exists( 'block-manifest' ) ? self::get_preset( 'block-manifest' ) : (object) array( 'blocks' => $blocks );
-		// manifest shouldn't remove default blocks...
 
-		$preset_blocks = isset( $preset_blocks_manifest->blocks ) ? (array) $preset_blocks_manifest->blocks : array() ;
+		$preset_blocks_manifest =  self::preset_exists( 'index' ) ? self::get_preset( 'index' ) : (object) array( 'blocks' => $blocks );
+		$preset_blocks = isset( $preset_blocks_manifest->production ) ? (array) $preset_blocks_manifest->production : array() ;
+
+		// todo: Remove next line because it adds the hard coded list of blocks... which makes.
```

Comment:
Something seems wrong in this comment.

Reply:
Seems to have been introduced by the rebase. I just amended the commit to fix.